### ISSUE TITLE
[BUG], M3u8, the sound and picture are not synchronized after dragging the video

### ISSUE THREAD

**Issue by devyueyue** [jhomlala/betterplayer#1084]
M3u8, the sound and picture are not synchronized after dragging the video.

After dragging, the display screen plays. There is no sound. After a few seconds, the sound appears, but the sound and the screen play are not synchronized

**Comment by RonakPustack** [comment]
I have been experiencing a similar issue, although I have seen this issue only in iOS, on android everything works perfectly.

**Comment by devyueyue** [comment]
Yes, it's just that the voice and picture are not synchronized on the ios phone, and the Android phone is normal

**Comment by devyueyue** [comment]
> 我一直遇到类似的问题，虽然我只在 iOS 上看到过这个问题，但在 android 上一切正常。

Have you solved this problem？

**Comment by Am607** [comment]
hey i have same issue any one is fixed the issue

**Comment by devyueyue** [comment]
> hey i have same issue any one is fixed the issue

Hello, do you have a solution, please send a link address, thank you
